src/js/themes/hpe.js

```diff
@@ -27,6 +27,30 @@ const deepFreeze = (obj) => {
   return Object.freeze(obj);
 };
 
+const primaryBackground = (props) => {
```

Comment:
Add summary comment about why we need it and if there's any direction on how to clean it up.